models/config_yaml_test.go

```diff
@@ -7,6 +7,44 @@ import (
 )
 
 var _ = Describe("ConfigYaml", func() {
+	Describe("NewCofigYaml", func() {
+		It("should define default value for forwarder", func() {
+			configYaml, _ := NewConfigYAML(`name: scheduler-name
+game: game
```

Comment:
I've seen that we basically copy the yaml for most test cases in maestro. Maybe having a parameterized function that returns a new yaml config would be less verbose (this comment is more for future commits, not necessarily you have to change this MR).